docs/sources/flow/reference/components/discovery.file.md

```diff
@@ -59,21 +59,37 @@ values.
 
 ## Examples
 
+### Send `/tmp/logs/*.log` files to Loki
+
 This example discovers all files and folders under `/tmp/logs`. The absolute paths are 
 used by `loki.source.file.files` targets.
 
 ```river
 discovery.file "tmp" {
-    path_targets = [{"__path__" = "/tmp/logs/**/*.log"}]
+  path_targets = [{"__path__" = "/tmp/logs/**/*.log"}]
 }
 
 loki.source.file "files" {
-    targets    = discovery.file.tmp.targets
-    forward_to = [ /* ... */ ]
+  targets    = discovery.file.tmp.targets
+  forward_to = [loki.write.endpoint.receiver]
+}
+
+loki.write "endpoint" {
+  endpoint {
+      url = LOKI_PATH
+      basic_auth {
+          username = USERNAME
+          password = PASSWORD
+      }
+  }
 }
 ```
+Replace the following:
+  - `LOKI_PATH`: The URL of the Loki server to send logs to.
+  - `USERNAME`: The username to use for authentication to the Loki API.
```

Comment:
nit: In the above we reference remote_write API but here we say Loki, do we want to change remote_write to Prometheus Compatible?